selfdrive/car/docs_definitions.py

```diff
@@ -115,15 +135,38 @@ def init(self, CP: car.CarParams, all_footnotes: Dict[Enum, int]):
     else:
       self.tier = Tier.BRONZE
 
+    self.year_list = get_year_list(self.years)
+    self.detail_sentence = self.get_detail_sentence(CP)
+
     return self
 
+  def get_detail_sentence(self, CP):
+    if CP.notCar:
+      return "The body is a robotics dev kit that can run openpilot. <a href='https://www.commabody.com'>Learn more.</a>"
```

Comment:
this isn't necessarily true for a notcar. add an "if body" and raise an exception if there's any notcar's that aren't handled